packages/bpk-component-chip-group/src/BpkChipGroup.module.scss

```diff
@@ -0,0 +1,90 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import '../../bpk-mixins/index.scss';
+
+.bpk-chip-group-container {
+  display: flex;
+  align-items: center;
+  white-space: nowrap;
+}
+
+.bpk-chip-group {
+  display: flex;
+  gap: bpk-spacing-md();
+
+  &--rail {
+    flex-wrap: nowrap;
+  }
+
+  &--wrap {
+    flex-wrap: wrap;
+  }
+}
+
+.bpk-sticky-chip-container {
+  margin-inline-end: bpk-spacing-md();
+  padding-inline-end: bpk-spacing-md();
+
+  @include bpk-border-right-sm($bpk-line-day);
+
+  @include bpk-rtl {
+    @include bpk-border-left-sm($bpk-line-day);
+  }
+
+  &--on-dark {
+    @include bpk-border-right-sm($bpk-line-on-dark-day);
+
+    @include bpk-rtl {
+      @include bpk-border-left-sm($bpk-line-on-dark-day);
+    }
+  }
+}
+
+@include bpk-breakpoint-tablet {
+  // TODO: Should this be a new chip component type without text?
+  .bpk-sticky-chip {
+    padding-inline-end: 0;
+    padding-inline-start: bpk-spacing-md();
+
+    // TODO: better way to do this? Hide all contents of the chip except the leading icon.
+    span:not(:nth-of-type(1)) {
+      display: none;
+    }
+  }
```

Comment:
This was needed to implement the sticky chip only showing text on mobile. It seems a little hacky and made me think a new type of chip for this would be better. What do you think?

<img width="91" alt="image" src="https://github.com/Skyscanner/backpack/assets/22656572/af849d86-f3ae-4c86-8782-8390c4e0b63c">

Reply:
Feels a little weird, I'm wondering on an a11y perspective if this is problematic at all? Feels like an icon only chip or something like that 🤷

Reply:
I've created this ticket to add an icon only chip: https://skyscanner.atlassian.net/browse/DTO-5126